### ISSUE TITLE
Make snapshot function capable of data selection and restoring scene setup

### ISSUE THREAD

**Issue by kitchoi** [simphony/simphony-mayavi#132]
Assume Mayavi 4.4.3.

Possible implementations:

Signature:

```
snapshot(cuds, filename, data=None, restore_visualisation=None)
```

Usage:

```
snapshot(cuds=particles, filename="saved.png",
                data=("TEMPERATURE", "VELOCITY"), 
                restore_visualisation="saved_scene.mv2")    # saved_scene.mv2 is previously saved
```

**Comment by kitchoi** [comment]
Discussion should go to the wiki page or the PR#136.  Closing.

**Comment by kitchoi** [comment]
This PR is related and is merged #145
